apps/mobile-alphabet/src/app/Components/alphabet-card-detail.screen.tsx

```diff
@@ -1,11 +1,14 @@
 import React, { useEffect, useState } from 'react';
 import { Button, Image, Text, View } from 'react-native';
+import Sound from 'react-native-sound';
 import config from './Config.json';
 import { AlphabetData } from './Menu';
 
 export function AlphabetCardDetailScreen() {
     const [alphabetData, setAlphabetData] = useState<AlphabetData | null>(null);
 
+    const [audioLoaded, setAudioLoaded] = useState(false);
```

Comment:
Also, note that technically there is one such loading state for each audio file. I'm missing where you combine these into a single flag. We should take a careful look at this.